lib/check.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  calculateVersionsForEachDependency,
+  calculateMismatchingVersions,
+  filterOutIgnoredDependencies,
+  fixMismatchingVersions,
+  DependencyAndVersions,
+} from './dependency-versions.js';
+import { getPackages } from './workspace.js';
+
+/**
+ * Checks for inconsistencies across a workspace. Optionally fixes them.
+ * @param path - path to the workspace root
```

Comment:
It seems unclear what the `path` is relative to.

Reply:
Could be an absolute path or relative to the where the command/function is being run from.